lib/async_hooks.js

```diff
@@ -168,14 +169,16 @@ class AsyncResource {
     const asyncId = this[async_id_symbol];
     emitBefore(asyncId, this[trigger_async_id_symbol]);
     try {
-      return Reflect.apply(fn, thisArg, args);
+      return ReflectApply(fn, thisArg, args);
```

Comment:
Is it significantly slower with the property lookup? I'd prefer if we could avoid this change

Reply:
It improved it by about 3-5% locally. I have an alternative change that will work.